project/AppDependencies.scala

```diff
@@ -1,22 +1,20 @@
 import play.core.PlayVersion.current
-import play.sbt.PlayImport._
-import sbt.Keys.libraryDependencies
 import sbt._
 
 object AppDependencies {
 
   val compile = Seq(
-
-    "uk.gov.hmrc"             %% "bootstrap-play-26"        % "0.42.0",
-    "uk.gov.hmrc"             %% "stub-data-generator"      % "0.5.3"
+    "uk.gov.hmrc"  %% "bootstrap-play-26"        % "0.42.0",
+    "uk.gov.hmrc"  %% "stub-data-generator"      % "0.5.3",
+    "org.julienrf" %% "play-json-derived-codecs" % "3.3"
```

Comment:
are you using this anywhere?

Reply:
they're redundant imports now  which I used for some model case classes which have been deleted.

Reply:
removed redundant dep